arcgis-ios-sdk-samples/Route and directions/Display device location with NMEA data sources/DisplayDeviceLocationWithNMEADataSources.storyboard

```diff
@@ -0,0 +1,103 @@
+                    <view key="view" contentMode="scaleToFill" id="xJd-vV-Dsb">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
+                            <view clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Nk8-cN-fxr" customClass="AGSMapView">
+                                <rect key="frame" x="0.0" y="44" width="414" height="769"/>
+                            </view>
+                            <view contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" translatesAutoresizingMaskIntoConstraints="NO" id="8tL-4E-K67" userLabel="Status View">
+                                <rect key="frame" x="0.0" y="44" width="414" height="20.5"/>
+                                <subviews>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Satellites info will be shown here." lineBreakMode="wordWrap" numberOfLines="4" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="AuA-N5-5HY">
+                                        <rect key="frame" x="20" y="0.0" width="374" height="20.5"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                </subviews>
+                                <viewLayoutGuide key="safeArea" id="AxU-Fm-eyW"/>
+                                <color key="backgroundColor" name="statusLabelBackgroudColor"/>
+                                <constraints>
+                                    <constraint firstItem="AuA-N5-5HY" firstAttribute="trailing" secondItem="8tL-4E-K67" secondAttribute="trailingMargin" id="KNU-0K-aM8"/>
+                                    <constraint firstItem="AuA-N5-5HY" firstAttribute="leading" secondItem="8tL-4E-K67" secondAttribute="leadingMargin" id="WqK-hR-BZJ"/>
+                                    <constraint firstItem="AuA-N5-5HY" firstAttribute="bottom" secondItem="8tL-4E-K67" secondAttribute="bottom" id="cEp-ea-ZbS"/>
+                                    <constraint firstItem="AuA-N5-5HY" firstAttribute="top" secondItem="8tL-4E-K67" secondAttribute="top" id="hGl-Zo-qSF"/>
+                                </constraints>
+                            </view>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="0Y8-SV-RLM">
```

Comment:
This view has duplicate bottom constraints:

![image](https://user-images.githubusercontent.com/2257493/109368819-3dd43900-784f-11eb-93fe-6f5e4d2e24c3.png)

Please remove one of them.